### PR TITLE
update install steps for Amazon Linux 2

### LINKED ISSUE #207: Amazon Linux 2 installation guide incomplete (missing container-selinux)
It seems provided installation steps for Amazon Linux 2 are incomplete.

After adding kubic repo and running `yum install podman` it throws message about missing `container-selinux` dependency:

```
Loaded plugins: extras_suggestions, langpacks, priorities, update-motd
devel_kubic_libcontainers_stable                                                                                                                                                     | 1.3 kB  00:00:00
devel_kubic_libcontainers_stable/primary                                                                                                                                             | 9.8 kB  00:00:00
devel_kubic_libcontainers_stable                                                                                                                                                                      27/27
2 packages excluded due to repository priority protections
Resolving Dependencies
--> Running transaction check
---> Package podman.x86_64 0:1.8.1-2.1.el7 will be installed
--> Processing Dependency: podman-plugins = 1.8.1-2.1.el7 for package: podman-1.8.1-2.1.el7.x86_64
--> Processing Dependency: slirp4netns >= 0.3.0-2 for package: podman-1.8.1-2.1.el7.x86_64
--> Processing Dependency: containernetworking-plugins >= 0.7.5-1 for package: podman-1.8.1-2.1.el7.x86_64
--> Processing Dependency: runc for package: podman-1.8.1-2.1.el7.x86_64
--> Processing Dependency: nftables for package: podman-1.8.1-2.1.el7.x86_64
--> Processing Dependency: containers-common for package: podman-1.8.1-2.1.el7.x86_64
--> Processing Dependency: container-selinux for package: podman-1.8.1-2.1.el7.x86_64
--> Processing Dependency: conmon for package: podman-1.8.1-2.1.el7.x86_64
--> Running transaction check
---> Package conmon.x86_64 2:2.0.11-1.1.el7 will be installed
---> Package containernetworking-plugins.x86_64 0:0.8.5-11.1.el7 will be installed
---> Package containers-common.x86_64 2:0.1.41-7.1.el7 will be installed
---> Package nftables.x86_64 1:0.9.0-14.amzn2.0.1 will be installed
--> Processing Dependency: libnftnl.so.11(LIBNFTNL_11)(64bit) for package: 1:nftables-0.9.0-14.amzn2.0.1.x86_64
--> Processing Dependency: libnftnl.so.11()(64bit) for package: 1:nftables-0.9.0-14.amzn2.0.1.x86_64
---> Package podman.x86_64 0:1.8.1-2.1.el7 will be installed
--> Processing Dependency: container-selinux for package: podman-1.8.1-2.1.el7.x86_64
---> Package podman-plugins.x86_64 0:1.8.1-2.1.el7 will be installed
---> Package runc.x86_64 0:1.0.0-0.1.20190510.git2b18fe1.amzn2 will be installed
---> Package slirp4netns.x86_64 0:0.4.3-22.1.el7 will be installed
--> Running transaction check
---> Package libnftnl.x86_64 0:1.1.1-4.amzn2 will be installed
---> Package podman.x86_64 0:1.8.1-2.1.el7 will be installed
--> Processing Dependency: container-selinux for package: podman-1.8.1-2.1.el7.x86_64
--> Finished Dependency Resolution
Error: Package: podman-1.8.1-2.1.el7.x86_64 (devel_kubic_libcontainers_stable)
           Requires: container-selinux
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@TomSweeneyRedHat PTAL

cc: @pangorgo

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@TomSweeneyRedHat PTAL

File: getting-started/installation.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ provides updated packages for CentOS 7 which can be used unmodified on Amazon Li
 ```bash
 cd /etc/yum.repos.d/
 sudo wget https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/CentOS_7/devel:kubic:libcontainers:stable.repo
+sudo yum -y install yum-plugin-copr
+sudo yum -y copr enable lsm5/container-selinux
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md:47]: is lsm5 the appropriate landing spot for this?

Comment (lsm5) [reply getting-started/installation.md:47]: yes, there's already `rhcontainerbot/container-selinux` for centos8 which won't work for this.

**Comment by TomSweeneyRedHat** [comment]
OK, but at some point I'd like to get the lsm5 locations moved elsewhere.
LGTM

**Comment by lsm5** [comment]
> OK, but at some point I'd like to get the lsm5 locations moved elsewhere.
> LGTM

I hope Amazon Linux releases a rhel8 clone so we don't have to support this anymore. If you got any suggestions for a new location, i'm listening.

**Pull request merged**